jwt/src/main/scala/it/pagopa/interop/commons/jwt/package.scala

```diff
@@ -89,6 +91,46 @@ package object jwt {
   private def getInteropRoleClaimSafe(claims: JWTClaimsSet): Option[String] =
     Try(claims.getStringClaim(roleClaim)).toOption.flatMap(Option(_))
 
+  def getExternalId(claims: JWTClaimsSet): (Option[String], Option[String]) = {
+
+    def externalIdFromOrganizationClaim(): Try[(Try[String], Try[String])] = for {
+      externalIdFromClaim <- Try(claims.getJSONObjectClaim(ORGANIZATION_EXTERNAL_ID_CLAIM))
+        .flatMap(nullable => Option(nullable).toTry(GenericError("External Id in context is not in valid format")))
+      origin = Try(externalIdFromClaim.get(ORGANIZATION_EXTERNAL_ID_ORIGIN))
```

Comment:
it's ok to flatMap because I expect both of them or none at all
```suggestion
      origin <- Try(externalIdFromClaim.get(ORGANIZATION_EXTERNAL_ID_ORIGIN))
```